Compiling failure on Windows

<!-- Please fill in the following information to report a problem with Cantera. If you have a question about using Cantera, please post it on our Google Users' Group (https://groups.google.com/forum/#!forum/cantera-users). Feature enhancements should be discussed in the dedicated Cantera enhancements repository (https://github.com/Cantera/enhancements/new/choose) -->

**Problem description**

Compiling with MSVC 14.2 on Windows 10 from MS Visual Studio 2019 gives the following error message:

```
cl /Fobuild\src\kinetics\ReactionData.obj /c src\kinetics\ReactionData.cpp /EHsc /MD /nologo /D_SCL_SECURE_NO_WARNINGS /D_CRT_SECURE_NO_WARNINGS /O2 /Zi /Fdbuild\src\kinetics\ReactionData.obj.pdb /W3 /DNDEBUG  /Iinclude /Iinclude\cantera\ext /Ibuild\src /IC:\Users\bww09001\Miniconda3\envs\cantera-dev\Library\include ReactionData.cpp C:\Users\bww09001\Documents\GitHub\cantera\include\cantera/kinetics/ReactionData.h(43): error C2039: 'log': is not a member of 'std' predefined C++ types (compiler internal)(340): note: see declaration of 'std' C:\Users\bww09001\Documents\GitHub\cantera\include\cantera/kinetics/ReactionData.h(43): error C3861: 'log': identifier not found
```

These changes were introduced in #982. Unfortunately, I'm not that familiar with compiling on Windows to know how to fix this. One would assume that the logarithm function would be part of a standard library... I'm also not sure why this wasn't caught by the CI suite.

**Steps to reproduce**

1. Compile the current `main` branch (908a2cc) on Windows with MSVC 14.2. It's unclear whether this is related to the specific version of the compiler, or the standard include directories, or what.

**System information**

- Cantera version: `main`
- OS: Windows 10
- MSVC 14.2, VS 2019

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -141,7 +141,7 @@ jobs:
           python3 -m pip install ruamel.yaml scons numpy cython sphinx\<4.0 \
           sphinxcontrib-katex sphinxcontrib-matlabdomain sphinxcontrib-doxylink
       - name: Build Cantera with documentation
-        run: python3 `which scons` build -j2 doxygen_docs=y sphinx_docs=y debug=n optimize=n
+        run: python3 `which scons` build -j2 doxygen_docs=y sphinx_docs=y debug=n optimize=n use_pch=n
       # The known_hosts key is generated with `ssh-keygen -F cantera.org` from a
       # machine that has previously logged in to cantera.org and trusts
       # that it logged in to the right machine

--- a/include/cantera/kinetics/ReactionData.h
+++ b/include/cantera/kinetics/ReactionData.h
@@ -11,6 +11,8 @@
 #ifndef CT_REACTIONDATA_H
 #define CT_REACTIONDATA_H
 
+#include "cantera/base/ct_defs.h"
+
 namespace Cantera
 {
 

